Source/WebCore/rendering/RenderTextControlMultiLine.cpp

```diff
@@ -49,9 +49,6 @@ RenderTextControlMultiLine::~RenderTextControlMultiLine()
 
 void RenderTextControlMultiLine::willBeDestroyed()
 {
-    if (textAreaElement().isConnected())
-        textAreaElement().rendererWillBeDestroyed();
-
     RenderTextControl::willBeDestroyed();
 }
```

Comment:
You can remove this method altogether from the cpp and the header file afaik, since RenderTextControlMultiline directly inherits from RenderTextControl.

Reply:
Good shout! Can do it.

Reply:
Done.